pkg/monitor/monitor/template.go

```diff
@@ -381,7 +381,7 @@ func addTlsConfig(pc *config.Config) {
 		// TODO support tiflash tls when it gets ready
 		if sconfig.JobName == "pd" || sconfig.JobName == "tidb" || sconfig.JobName == "tikv" ||
 			sconfig.JobName == "pump" || sconfig.JobName == "drainer" ||
-			sconfig.JobName == "tiflash" || sconfig.JobName == "tiflash-proxy" {
+			sconfig.JobName == "tiflash" || sconfig.JobName == "tiflash-proxy" || sconfig.JobName == "lightning" {
```

Comment:
The lightning chart does not support TLS yet.

Reply:
DBaaS will not use lightning chart, Just set scheme is all right.